x-pack/plugins/session_view/public/hooks/use_eui_theme.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { shade, useEuiTheme as useEuiThemeHook } from '@elastic/eui';
+import { euiLightVars, euiDarkVars } from '@kbn/ui-theme';
+import { useMemo } from 'react';
+
+type EuiThemeProps = Parameters<typeof useEuiThemeHook>;
+type ExtraEuiVars = {
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  euiColorVis6_asText: string;
```

Comment:
for some reason, `euiColorVis6_asText` doesn't exist on `euiLightVars`, but it's a valid color token on the design library.